go/libkb/userconfig.go

```diff
@@ -5,49 +5,20 @@ package libkb
 
 import (
 	"encoding/hex"
-	"strings"
 
+	"github.com/keybase/client/go/kbun"
 	keybase1 "github.com/keybase/client/go/protocol/keybase1"
 	jsonw "github.com/keybase/go-jsonw"
 )
 
 //==================================================================
 
-// NormalizedUsername is a username that has been normalized (toLowered)
-// and therefore will compare correctly against other normalized usernames.
-type NormalizedUsername string
+// TODO: Remove these aliases once everything outside of this repo
+// points to kbun.
```

Comment:
Why bother TODO? Anything wrong with type alias for life? I think it'd be nice if it were a ticket number rather than a floating TODO.

Reply:
Added ticket number.

Nothing wrong with aliases per se (e.g. KBFSLogFileName makes sense as a forever alias), but I think for most cases once they've outlived their usefulness they can be removed, if only to force new uses to use the 'real' type instead of adding a dependency on libkb.